src/components/molecules/form-field/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { DetailedHTMLProps, InputHTMLAttributes } from "react";
+
+type InputProps = DetailedHTMLProps<
+  InputHTMLAttributes<HTMLInputElement>,
+  HTMLInputElement
+>;
+
+export type FormFieldProps = {
+  label: string;
```

Comment:
Suggestion: If this is a global type/interface we should move it to a folder from `models/types/form-field.ts`